Tasks/CopyFilesOverSSHV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -20,20 +20,22 @@
   "loc.input.help.failOnEmptySource": "Fail if no matching files to be copied are found under the source folder.",
   "loc.input.label.flattenFolders": "Flatten folders",
   "loc.input.help.flattenFolders": "Flatten the folder structure and copy all files into the specified target folder on the remote machine.",
-  "loc.messages.UseDefaultPort": "Using port 22 which is the default for SSH since no port was specified.",
-  "loc.messages.SourceNotFolder": "Source folder has to be a valid folder path.",
+  "loc.messages.CheckLogForStdErr": "Check the build log for STDERR from the command.",
```

Comment:
I just sorted the messages.  The ones I added are `FailedOnFile` and `NumberFailed`.